ui/js/controllers/bugfiler.js

```diff
@@ -166,6 +168,11 @@ treeherder.controller('BugFilerCtrl', [
                 descriptionStrings += $scope.modalComment;
             }
 
+            var keywords = "";
+            if($scope.isIntermittent) {
+                keywords += "intermittent-failure";
```

Comment:
If there was more than one keyword, they'd have to be comma separated, right?

If so, seeing as we're appending to the string (presumably since we want to add support for other keywords later), we should probably include a trailing comma in the string being appended.